Pipfile

```diff
@@ -11,9 +11,10 @@ name = "pcic"
 [packages]
 python-dateutil = "*"
 cdo = "==1.5.3"
-nchelpers = "==5.5.6"
+nchelpers = "==5.5.8"
 Pint = "*"
 PyYAML = "*"
+dp = {editable = true, path = "."}
```

Comment:
I'm still not confident that specifying the package under testing within its own Pipfile is the right thing to do. But if it works, then barring better guidance I'm OK with it. We should read up further on `pipenv` and clarify whether this is an intended use case.